apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -173,6 +173,15 @@ const getNewMessageId = () => {
   return latestMessageId;
 };
 
+export const RESET_MODEL_NOTIFICATION: ChatCompletionMessage = {
+  id: getNewMessageId(),
+  role: Role.MODEL_UPDATE,
+  chatMessageTextBold: 'Model customizations and model card information',
```

Comment:
I find this a little confusing, since `chatMessageText` is the core message for every other message type. I think it'd be easier for future readers if the `chatMessageText` remained the main message (and is bolded when rendered in this case), and there was a `chatMessageSuffix` or something that contained the extra text.

Reply:
That makes sense - thanks! Will update.